rclcpp/include/rclcpp/timer.hpp

```diff
@@ -58,6 +58,15 @@ class TimerBase
   void
   cancel();
 
+  /// Return the timer cancellation state.
+  /**
+   * @return true if the timer has been cancelled, false otherwise
+   * @throw std::runtime_error if the state could not be read
+   */
+  RCLCPP_PUBLIC
+  bool
+  is_cancelled();
```

Comment:
Can you please use `is_canceled`? We don't use the British variants anywhere else in our code/documentation (e.g. we use behavior not behaviour), see:

https://www.dictionary.com/browse/canceled

Reply:
My mistake. Addressed in the latest commit.